src/main/java/seedu/duke/parsermanager/ParseAddProperty.java

```diff
@@ -106,7 +107,11 @@ public class ParseAddProperty extends ParseAdd {
 
     private static HashMap<String, String> UNIT_TYPE_HASHMAP;
 
-
+    private static final String HASH_SYMBOL = "#";
+    private static final String LANDED_PROPERTY_TAG = "LP";
+    public static final String HDB_TAG = "HDB";
+    public static final String CONDOMINIUM = "Condominium";
+    public static final String PENTHOUSE = "Penthouse";
```

Comment:
Would be good if u can partition these constants and insert meaningful comments :D